### DISCUSSION TITLE
Forgetting password

### DISCUSSION THREAD

**Discussion by BarakatElzoghly0** [community/community#148106] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I signed in to GitHub with an account and obtained the Student Pack for a specific username. Later, I forgot my password. From the available recovery options, I chose to unlink the username. 

Afterward, I signed in again with the same account but used a different username. When I applied for the Student Pack, I received the following message: 

> "{my email} has already been used to obtain an academic discount on GitHub account(s) {old username}."

Now, I want to obtain the academic discount on either my old or new username. What should I do?

**Comment by davevad93** [comment, marked as answer]
Hi @BarakatElzoghly0 ,

`Student Developer Pack` benefits are tied to the academic email used to apply not the username, if your account is using the same email you can't apply again because it's already listed on GitHub Education as "**already been used to obtain an academic discount**", besides if you use another account it's not possible to transfer student benefits from one account to another.

You can try to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and ask them if they can help you recover your benefits.

**Reply by BarakatElzoghly0** [reply]
Hi @davevad93 ,
I really appreciate it!
